blackbox/docs/installation.txt

```diff
@@ -94,7 +94,8 @@ additional packages if we see enough interest.
 
  :doc:`Configuring CrateDB <configuration>`
 
- :doc:`Hello CrateDB <hello>` - To learn how to interact with CrateDB.
+ `Hello CrateDB <https://crate.io/docs/crate/guide/tutorials/hello.html>`_ - To
```

Comment:
Could you change this link to `` `Hello CrateDB`_ `` and put at `` .. _`Hello CrateDB`: https://crate.io/docs/crate/guide/tutorials/hello.html `` at the bottom of the document, to be consistent with the other external links in the document?